radixdlt-java/src/main/java/com/radixdlt/client/core/network/jsonrpc/RadixLocalSystem.java

```diff
@@ -2,7 +2,7 @@
 
 import org.radix.serialization2.SerializerId2;
 
-@SerializerId2("LOCALSYSTEM")
+@SerializerId2("internal.local_system")
```

Comment:
Not sure why this would be in namespace internal?  Doesn't seem quite right.  Network or api maybe?